src/pages/dashboard/monitor/components/Charts/index.tsx

```diff
@@ -0,0 +1,15 @@
+import Gauge from './Gauge';
+import MiniArea from './MiniArea';
+import Pie from './Pie';
+import TagCloud from './TagCloud';
+import WaterWave from './WaterWave';
+
+const Charts = {
+  Pie,
+  WaterWave,
+  Gauge,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'Gauge' is not sorted alphabetically](https://app.codacy.com/app/afc163/ant-design-pro/pullRequest?prid=4583897)